microservices/projects/handlers/tasks_handlers.go

```diff
@@ -213,7 +214,60 @@ func (p *Projects) DeleteTask(w http.ResponseWriter, r *http.Request) {
 
 // EditTask edit the details of select task.
 func (p *Projects) EditTask(w http.ResponseWriter, r *http.Request) {
-	http.Error(w, "Not Implemented", http.StatusNotImplemented)
+	projectName := chi.URLParam(r, "project-name")
+	taskIDStr := chi.URLParam(r, "task-id")
```

Comment:
url param or put in payload? Up to you

Reply:
Does not really matter can be easily changed if we want later on.

Reply:
![image](https://user-images.githubusercontent.com/29551974/218282395-f6054202-348c-4142-8c2a-d5e662e66c7e.png)